packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -165,11 +170,31 @@ export class CucumberJSAllureFormatter extends Formatter {
       return;
     }
 
+    const doc = this.documentMap.get(pickle.uri);
+    const scenarioId = pickle?.astNodeIds?.[0];
+    const scenario = this.scenarioMap.get(scenarioId);
     this.testCaseStartedMap.set(data.id, data);
     this.testCaseTestStepsResults.set(data.id, []);
     this.currentTest = new AllureTest(this.allureRuntime, Date.now());
     this.currentTest.name = pickle.name;
 
+    this.currentTest?.addLabel(LabelName.LANGUAGE, "JavaScript");
+    this.currentTest?.addLabel(LabelName.FRAMEWORK, "CucumberJS");
+
+    if (doc && doc.feature) {
+      this.currentTest.addLabel(LabelName.FEATURE, doc.feature?.name);
```

Comment:
why `feature?` ?

Reply:
Because the current feature title locates inside the parsed document. So `doc.feature.name` field inside:

```gherkin
Feature: do something
```

Equals to `do something`